contracts/PodCastNFT.sol

```diff
@@ -130,33 +124,24 @@ contract PodCastNFT is ERC721URIStorage, Ownable, ConsensusAdminable {
 
     function mint(
         string memory _imageURI,
-        string memory _role,
-        string memory _roleInCommunity,
+        string[] memory _roles,
         string memory _point,
         address _to
-    ) public returns (uint256) {
-        require(
-            hasRole(ADMIN_ROLE, msg.sender),
-            "You are not authorized to mint"
-        );
+    ) public onlyOwner returns (uint256) {
         require(balanceOf(_to) == 0, "User has had already a memebrship NFT");
 
         _tokenIds.increment();
         uint256 newItemId = _tokenIds.current();
         _safeMint(_to, newItemId);
-
-        _communityMemberShip[newItemId] = isCommunityMemberByCommunityRole(
-            _roleInCommunity
-        );
+        roles[_to] = _roles;
```

Comment:
It might be better to implement this with setRoles (or setRolesWithAdmin). This is a short process, but it avoids duplication of code for permission checks, etc.